Brainduino/Brain.cpp

```diff
@@ -7,28 +7,28 @@
 
 #include "Brain.h"
 
-Brain::Brain(Print *printer, Stream *streamIn, BrainDelegate *delegate)
+Brain::Brain(Print *printer, Stream *stream_in, BrainDelegate *delegate)
 {
-    set(printer, streamIn, delegate, nullptr);
+    set(printer, stream_in, delegate, nullptr);
 }
 
-Brain::Brain(Print *printer, Stream *streamIn, BrainDelegate *delegate,
+Brain::Brain(Print *printer, Stream *stream_in, BrainDelegate *delegate,
              char const *code)
 {
-    set(printer, streamIn, delegate, code);
+    set(printer, stream_in, delegate, code);
 }
 
-void Brain::set(Print *printer, Stream *streamIn, BrainDelegate *delegate,
+void Brain::set(Print *printer, Stream *stream_in, BrainDelegate *delegate,
                 char const *code)
 {
     reset();
     _printer = printer;
-    _streamIn = streamIn;
+    _streamIn = stream_in;
```

Comment:
I imagine that you will have a problem in this line. Could you rename the variable _streamIn?